### PR TITLE
New Resource: azurerm_application_insights_workbook_template

### PR DESCRIPTION
**Pull request by ms-zhenhua**
[Swagger Link](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20)
image

### CODE REVIEW

File: internal/services/applicationinsights/validate/workbook_template_id.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package validate
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-azure-sdk/resource-manager/applicationinsights/2020-11-20/applicationinsights"
+)
+
+func WorkbookTemplateID(input interface{}, key string) (warnings []string, errors []error) {
```
Comment (tombuildsstuff) [inline internal/services/applicationinsights/validate/workbook_template_id.go:9]: this is available within the sdk (`applicationinsights.ValidateWorkbookTemplateID`) - so we can remove this?

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
A couple of minor comments but this otherwise LGTM 👍

**Comment by ms-zhenhua** [comment]
Hi @tombuildsstuff, thank you for reviewing my code. I have resolved the comments, kindly help review it again. Thanks.

**Review by katbyte — APPROVED** [review]
LGTM 🎩

**Pull request merged**